pkg/kubectl/resource_printer.go

```diff
@@ -60,6 +60,9 @@ const (
 	loadBalancerWidth = 16
 )
 
+// This indicates that a particular StorageClass nominates itself as the system default.
+const betaAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
```

Comment:
@eparis - how about this, for now we can use exported const in the types.go from the storage package, that way we have a single place to maintain these and they are easily accessible...